File: PrjectHejer.DataAccess/Models/Customer.cs
using System;
using System.Collections.Generic;

namespace PrjectHejer.DataAccess.Models;

public partial class Customer
{
    public Guid CustomerId { get; set; }

    public string Name { get; set; } = null!;

    public string? Email { get; set; }

    public string? Phone { get; set; }

    public DateTime CreatedAt { get; set; }
}

File: PrjectHejer.Client/src/types/customer.ts
export interface Customer {
  customerId: string;
  name: string;
  email: string;
  phone: string;
}

export interface SingleCustomer {
  CustomerId: string;
  Name: string;
  Email: string;
  Phone: string;
  Images: Array<{ id: string; base64Image: string }>;
}
